user/pages/04.Reference/05.object-storage/docs.en.md

```diff
@@ -40,7 +40,7 @@ Setting bucket/object ACLs is only partially possible and currently unintuitive
 ## Buckets
 
 Buckets are the logical unit SysEleven Stack Object Storage uses to stores objects.
-Every bucket name is unique within a SysEleven Stack region.
+Every bucket name must be unique within a SysEleven Stack region.
```

Comment:
As this fact cannot be circumvented accidentally or intensionally, it is just a fact. Every bucket name *is* unique.

Reply:
its not only a fact, it is also a requirement. the latter is more important to know to customers in my opinion.

Reply:
Exactly. In my view, the old version seems to say (or somebody could understand it like that) that the system somehow magically makes this true, and the customer doesn't have to do anything.

Reply:
Actually, then, wo should give really helpful advice to our customers.
>Every bucket name can only be used once per SysEleven Stack region among all customers. If a desired bucket name is already taken by any customer, you cannot create another bucket with that name. To minimize the risk of collisions, avoid common and short words, combine multiple rare words, prefix them with your company or brand name, add random numbers or hashes (e.g. `syseleven-bucketname-example-7hg3x` instead of `test` is more likely to succeed). If you create buckets automatedly, prepare to retry with a different choice or resort to manual action.

I don't know if we want or need a dispute policy here, but we might offer to gently contact offenders that occupy bucket-names that contain other customers' brand names. But I'd rather not give any legal promises.

Reply:
I like that text!